src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -437,15 +445,30 @@ export default class SourceFileLinter {
 		}
 
 		if (this.isSymbolOfPseudoType(symbol)) {
-			this.#reporter.addMessage({
-				node: moduleSpecifierNode,
-				severity: LintMessageSeverity.Error,
-				ruleId: "ui5-linter-no-pseudo-modules",
-				message:
-					`Import of pseudo module ` +
-					`'${moduleSpecifierNode.text}'`,
-				messageDetails: "Import library and reuse the enum from there",
-			});
+			const moduleNamespaceName = moduleSpecifierNode.text.replaceAll("/", ".");
+			const isDataType = !!this.#dataTypes[moduleNamespaceName];
+			if (isDataType) {
+				this.#reporter.addMessage({
+					node: moduleSpecifierNode,
+					severity: LintMessageSeverity.Error,
+					ruleId: "ui5-linter-no-pseudo-modules",
+					message: `Deprecated access to DataType ('${moduleSpecifierNode.text}').`,
+					messageDetails: `DataType can be accessed only via the static ` +
+					`DataType.getType("${moduleNamespaceName}") method. ` +
+					"{@link topic:00737d6c1b864dc3ab72ef56611491c4 Migrating Deprecated Pseudo Modules}",
```

Comment:
This section is currently not part of the 1.120 SDK, see https://ui5.sap.com/1.120/#/topic/00737d6c1b864dc3ab72ef56611491c4

Reply:
I was wondering wether the links should point to 1.120 which seems logical to me as we use 1.120 types. 
On the other side all the other links, that are comming from types' descriptions are pointing to the latest version...

What should be the solution then ?

Reply:
Topics would be always served from the latest version https://github.com/SAP/ui5-linter/pull/112/commits/5419063c84acb0b36aff8417c882df62584ffaaa